packages/venia-ui/lib/components/Header/header.css

```diff
@@ -39,6 +39,9 @@
 .primaryActions {
     grid-area: primary;
     justify-self: start;
+    display: grid;
+    grid-template-columns: 1fr 1fr;
```

Comment:
I think `auto` is the better size for these columns, and it's okay for the number of columns to be implicit here, so I think this declaration can be safely removed.
```suggestion
```

Reply:
Just removing causes them to stack into rows.
![2020-06-11 10 54 45](https://user-images.githubusercontent.com/1774374/84408641-ff6bf080-abd1-11ea-85e8-1d8ad9c1ad97.gif)

I also added `grid-auto-flow: column;` to resolve this :)

Reply:
Good call. 👍